src/platform/user/profile/utilities/index.js

```diff
@@ -131,7 +131,8 @@ export function mapRawUserDataToState(json) {
 // as a trigger to properly update any components that subscribe to it.
 export const hasSession = () => localStorage.getItem('hasSession');
 
-export const hasSessionSSO = () => localStorage.getItem('hasSessionSSO');
+export const hasSessionSSO = () =>
+  localStorage.getItem('hasSessionSSO') === 'true';
```

Comment:
Dumb question: Is `true` supposed to be a string or a boolean? I only ask because [it's a boolean in your test](https://github.com/department-of-veterans-affairs/vets-website/pull/11963/files#diff-cabf259768d532bf52ac329b447e537bR11).

Reply:
good point, I could have been more explicit in the test. 

localStorage values are coerced to strings, so `setItem` with a `true` will get converted to "true". I'll just make it "true" in the test, too, to avoid confusion